bin/dfx-stock-deploy

```diff
@@ -38,11 +38,11 @@ dfx-sns-demo --network "$DFX_NETWORK" --ic_commit "$DFX_IC_COMMIT" --ic_dir "$IC
 dfx-software-mock-bitcoin-install --network "$DFX_NETWORK" --ic_commit "$DFX_IC_COMMIT"
 
 : Add 1 open SNS project.
-dfx-sns-demo-mksns --network "$DFX_NETWORK" --confirmation_text "I confirm the confirmation text"
+dfx-sns-demo-mksns --network "$DFX_NETWORK" --confirmation_text "I confirm the confirmation text" --index-offset 0
```

Comment:
Shouldn't this have index 1 because an SNS has already been made in `dfx-sns-demo` on line 36 of `dfx-stock-deploy`?

https://github.com/dfinity/snsdemo/blob/2d1d5aa67b89edddde1cc787b61cf5fbf0b344be/bin/dfx-sns-demo#L82

Reply:
Correct. Updated.